src/main/java/com/aloc/aloc/problem/service/ProblemSolvingService.java

```diff
@@ -61,25 +61,28 @@ public String checkAndUpdateProblemSolved(ProblemResponseDto problem, User user)
 		}
 	}
 
-	private int calculateCoinToAdd(Long problemId) {
-		// 3등까지는 50코인, 4등부터는 30코인을 지급합니다.
-		long solvedUserCount = userProblemRepository.countSolvingUsersByProblemId(problemId, currentSeason);
-		return solvedUserCount <= MAX_SOLVED_USER_COUNT_FOR_BONUS ? COINS_FOR_TOP_SOLVERS : COINS_FOR_OTHER_SOLVERS;
+	public List<UserProblem> getSolvedUserListByProblemId(Long problemId) {
+		return userProblemRepository.findAllByProblemIdAndIsSolvedIsTrue(problemId);
```

Comment:
```suggestion
		return userProblemRepository.findAllSolvedProblemByProblemId(problemId);
```

이렇게 함수명을 변경하는건 어떨까요?
작성하신 함수명에서 problemId만 넘기는데 AndIsSolvedIsTrue가 있어서 의견남겨봐요~!

Reply:
jpa 문법 때문에 저 형식으로 정의되어있는 것입니다! ㅎㅎ
isSolved값이 true이고 problemId가 problemId인 값들을 불러오는 함수입니다!